sysl2/sysl/swagger/tests/EXAMPLE_SWAGGER_SPEC_WITH_ENDPOINT_PATH_WITH_DEFAULT_RESPONSE.sysl

```diff
@@ -0,0 +1,21 @@
+##########################################
+##                                      ##
+##  AUTOGENERATED CODE -- DO NOT EDIT!  ##
+##                                      ##
+##########################################
+
+testapp "Goat CRUD API" [package="package_foo"]:
+    @version = "1.2.3"
+    @host = "goat.example.com"
+    @description =:
+        | No description.
+
+    /api/v1:
+
+        /goat/status:
+            GET:
+                | Check goat status
+                return default
```

Comment:
IMO this should be just `return`

Reply:
This is exactly what the python generated... LEts turn this into a ticket.